src/Private/Console.ps1

```diff
@@ -0,0 +1,987 @@
+
+<#
+.SYNOPSIS
+    Displays Pode server information on the console, including version, PID, status, endpoints, and control commands.
+
+.DESCRIPTION
+    The Show-PodeConsoleInfo function displays key information about the current Pode server instance.
+    It optionally clears the console before displaying server details such as version, process ID (PID), and running status.
+    If the server is running, it also displays information about active endpoints and OpenAPI definitions.
+    Additionally, it provides server control commands like restart, suspend.
+
+.PARAMETER ClearHost
+    Clears the console screen before displaying server information.
+
+.NOTES
+    This is an internal function and may change in future releases of Pode.
+#>
+function Show-PodeConsoleInfo {
+    param(
+        [switch]
+        $ClearHost,
+
+        [switch]
+        $Force,
+
+        [switch]
+        $ShowTopSeparator
+    )
+
+
+
+    # Get the current server state and timestamp
+    $serverState = Get-PodeServerState
+    $timestamp = if ($PodeContext.Server.Console.ShowTimeStamp ) { "[$(Get-Date -Format 'yyyy-MM-dd HH:mm:ss')]" } else { '' }
+
+    if (!$PodeContext) { return }
+
+    # Define color variables with fallback
+    $headerColor = if ($null -ne $PodeContext.Server.Console.Colors.Header) {
+        $PodeContext.Server.Console.Colors.Header
+    }
+    else {
+        [System.ConsoleColor]::White
+    }
```

Comment:
`Protect-PodeValue` here?

Reply:
to be honest I can remove this protection is not possible to use it without `$PodeContext`